shared/middleware/files_api.rb

```diff
@@ -567,7 +567,7 @@ def files_put_file(encrypted_channel_id, filename, body)
     new_entry_hash.to_json
   end
 
-  # POST /v3/files/<channel-id>/?version=<version-id>&project_version=<project-version-id>
+  # POST /v3/files/<channel-id>/?version=<version-id>&files-version=<project-version-id>
```

Comment:
Are these `&files-version` documentation changes accurate?  I don't see this elsewhere in your diff.

Reply:
Yes, this is a correction to the docs. There is no such thing as `project_version`, so I fixed it.

Reply:
:+1: Excellent.